src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -110,6 +128,24 @@ static ClusterStateMachine()
                 StartIndex = 0,
                 Count = 1, // Database, Separator, Commands count
             });
+
+            // We use the follow format for the certificates data
+            // { key, hash, data }
```

Comment:
```suggestion
            // { thumbprint, public key hash, data }
```

Isn't that the case?

Reply:
key & hash are too generic

Reply:
ok